src/modules/updatedom.js

```diff
@@ -0,0 +1,31 @@
+import { localData } from './globals.js';
+
+const updateDom = () => {
+  const data = localData;
+  const list = document.querySelectorAll('.to-do-item');
+
+  list.forEach((listItem, i) => {
+    listItem.setAttribute('id', data[i].index);
```

Comment:
This line is throwing errors to the browser console. Also, it's not changing the id of the element.
Kindly change it by this method: 

`document.getElementById("demo").id = "newid";`

Reply:
I'm not seeing any error regarding that piece of code. Can you give a screenshot?

Reply:
@PrantoshB , Please don't use the live link from the readme. I haven't updated the link there yet. Please use this link- https://portfolio.utsargo.tech/best-practices-exercise/

Reply:
Sorry that the GIF is not showing the whole problem.

Here's the video:


https://user-images.githubusercontent.com/93311467/207395884-cdb58fbe-e310-4a95-94d3-be0501727248.mp4

And for the `id` issue, I meant to say: when you inspect the list items, it shows the same id for every item. So, it's not changing accordingly.

Now the console error is gone for me also. It can be because of some other issues.

Cheers 😉